### PR TITLE
Feature/410/implement video presentation block on tutor profile

### PR DESCRIPTION
**Pull request by Radomyr-kh**
videoPresentation

Implemented a mock block 'Video Presentation' on "Tutor's profile"

### CODE REVIEW

File: src/pages/tutor-profile/TutorProfile.jsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import Container from '@mui/material/Container'
 import CompleteProfileBlock from '~/components/complete-profile/CompleteProfileBlock'
 import { profileItems } from '~/components/profile-item/complete-profile.constants'
+import VideoPresentation from '~/containers/tutor-profile-page/video-presentation/VideoPresentation'
 
 const TutorProfile = () => {
   return (
-    <Container data-testid='tutorHome' style={ { flex: 1 } }>
-      TutorProfile Page Placeholder
+    <Container data-testid='tutorProfile' style={ { flex: 1, maxWidth: '1128px', margin: '0 auto' } }>
```
Comment (MickhaelSh) [inline src/pages/tutor-profile/TutorProfile.jsx:8]: maxWidth: '1128px', margin: '0 auto' is no needed for container

File: src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from 'react-i18next'
+import { Box, Typography } from '@mui/material'
```
Comment (Yurenko) [inline src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx:2]: Use imports like this, for all mui components:
import Box from '@mui/material/Box'

File: src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import VideoBox from '~/components/video-box/VideoBox'
+import videoImg from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
+
+
+const VideoPresentation = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Box className='section' data-testid='videoPresentation' sx={ { flexDirection: 'column' } }>
```
Comment (Tolik170) [inline src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx:11]: you don`t use this data-testid, so you can remove it

File: src/pages/tutor-profile/TutorProfile.jsx (hunk 1 above)
Comment (Tolik170) [reply src/pages/tutor-profile/TutorProfile.jsx:8]: also, remove this data-testid

Comment (Radomyr-kh) [reply src/pages/tutor-profile/TutorProfile.jsx:8]: > maxWidth: '1128px', margin: '0 auto' is no needed for container

without _maxWidth_ being set here, my VideoBox Component doesn't render properly

File: src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx (hunk 2 above)
Comment (Radomyr-kh) [reply src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx]: fixed

File: src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx (hunk 3 above)
Comment (Radomyr-kh) [reply src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx]: fixed

File: src/pages/tutor-profile/TutorProfile.jsx (hunk 1 above)
Comment (Tolik170) [reply src/pages/tutor-profile/TutorProfile.jsx]: > > maxWidth: '1128px', margin: '0 auto' is no needed for container
> 
> without _maxWidth_ being set here, my VideoBox Component doesn't render properly

why? It should be adaptive

Comment (Radomyr-kh) [reply src/pages/tutor-profile/TutorProfile.jsx]: Fixed

**Review by Tolik170 — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Pull request closed without merging**